src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -158,15 +159,30 @@ final case class PurposeApiServiceImpl(
     val operationLabel = s"Retrieving Purpose $id"
     logger.info(operationLabel)
 
+    def tryToValidateRiskAnalysisForm(
+      riskAnalysisForm: Option[PurposeManagementDependency.RiskAnalysisForm]
+    ): Future[Option[Boolean]] = {
+      Future.successful(
+        riskAnalysisForm
+          .map(RiskAnalysisConverter.dependencyToApi(_))
+          .map(
+            RiskAnalysisValidation
+              .validate(_, schemaOnlyValidation = false)
+              .isValid
+          )
+      )
+    }
```

Comment:
Stefano was suggesting to return the result not incapsulated in `Future` (and we can avoid the `Option` too)
```suggestion
    ): Boolean = 
        riskAnalysisForm
          .map(RiskAnalysisConverter.dependencyToApi(_))
          .map(
            RiskAnalysisValidation
              .validate(_, schemaOnlyValidation = false)
              .isValid
          )
          .getOrElse(false)   
```